README.md

```diff
@@ -152,3 +152,54 @@ This is the template PVC. No namespace is supplied since the PVC is excpected to
 1. Monitor the importer pod:
 
         $ kubectl logs <unique-name-of-importer pod>  # shown in controller log above
+
+### Security Configurations
```

Comment:
I'd like to see this section listed in the ToC

Reply:
ack